http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1927,16 +1880,40 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-  /proxy/invalidateToken:
+  "/proxy/setCookie":
```

Comment:
is this runtime specific or applicable to both runtime and galaxy?

Reply:
Yep, applicable to both runtime and galaxy